pythonforandroid/recipes/libxml2/__init__.py

```diff
@@ -13,10 +13,15 @@ class Libxml2Recipe(Recipe):
 
     def get_recipe_env(self, arch):
         env = super(Libxml2Recipe, self).get_recipe_env(arch)
+        return env
 
-        sh.sed('runtest$(EXEEXT) \/ \/', 'Makefile')
-        sh.sed('testrecurse$(EXEEXT)$//', 'Makefile')
```

Comment:
@inclement it is having error here(with this recipe alone others are working fine ( I have not tested bs4 recipe as it depends on this one)).